### PR TITLE
Review and merge forest episode in German branch

### PR DESCRIPTION
**Pull request by cojades**
@casey-collab please merge forest episode into master

### CODE REVIEW

File: german/schwimmen/gefahr/wald/flucht.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Sie rennen in Panik los. Dabei stolpern Sie über das dichte Wurzelwerk und fallen hin.
```
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/flucht.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: german/schwimmen/gefahr/wald/klettern.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Vom Adrenalinstoß mit großen Kräften versehen ziehen Sie sich elegant am dichten
```
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/klettern.md]: This file has also been added to a directory that already contained a page.

File: german/schwimmen/gefahr/wald/wald.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/wald.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cojades** [comment]
@casey-collab  
Added directories as suggested

File: german/schwimmen/gefahr/wald/flucht.md (hunk 1 above)
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/flucht.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: german/schwimmen/gefahr/wald/klettern.md (hunk 2 above)
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/klettern.md]: This file has also been added to a directory that already contained a page.

File: german/schwimmen/gefahr/wald/wald.md (hunk 3 above)
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/wald.md]: This file has also been added to a directory that already contained a page.

File: german/schwimmen/gefahr/wald/flucht.md (hunk 1 above)
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/flucht.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: german/schwimmen/gefahr/wald/klettern.md (hunk 2 above)
Comment (casey-collab) [inline german/schwimmen/gefahr/wald/klettern.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: german/schwimmen/gefahr/wald/flucht.md (hunk 1 above)
Comment (cojades) [inline german/schwimmen/gefahr/wald/flucht.md]: @casey-collab

Added directories as suggested

File: german/schwimmen/gefahr/wald/klettern.md (hunk 2 above)
Comment (cojades) [inline german/schwimmen/gefahr/wald/klettern.md]: @casey-collab

Added directories as suggested

File: german/schwimmen/gefahr/wald/wald.md (hunk 3 above)
Comment (cojades) [inline german/schwimmen/gefahr/wald/wald.md]: @casey-collab

Added directories as suggested

**Comment by cojades** [comment]
@casey-collab

Added directories as suggested

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**